File: Ruby/mathematics.rb
result = gets.chomp.to_i
expr = gets.chomp.split(" ")
check = ''
#=begin
for a in expr
  if eval(a) == result
    check = 'ok'
    break
  end
end

if check == 'ok'
  puts "index #{expr.index(a)}"
else
  print('none')
end

File: Python/mathematics.py
result = int(input())
expr = input().split(" ")
check = ''

for a in expr:
  if eval(str(a)) == result:
    check = 'ok'
    break

if check == 'ok':
  print(f"index {expr.index(a)}")
else:
  print('none')